### PR TITLE
addons: add doc for addon template

### PR DESCRIPTION
**Pull request by CvH**
@lrusak pls have a look and hit the button if okay - the gang had already a look

### CODE REVIEW

**Comment by lrusak** [comment]
I'm happy with this, it would be nice to add docs for the `addon()` function.

File: packages/addons/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#Addon skeleton
+This is an example how the default structure of an addon should look like.
+Please be aware that this is an moving target and could be changed to adapt changes at the build system.
+
+Lines labelled with `#` are completely optionally (beside the license) and should be omitted if you don't need them.
```
Comment (chewitt) [inline packages/addons/README.md]: Except for the license header, any lines prefixed with `#` are optional and may be omitted if not required.

File: packages/addons/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#Addon skeleton
+This is an example how the default structure of an addon should look like.
+Please be aware that this is an moving target and could be changed to adapt changes at the build system.
```
Comment (chewitt) [inline packages/addons/README.md]: Remove this line. If the structure changes this skeleton should be updated to reflect the changes.

File: packages/addons/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+#Addon skeleton
```
Comment (chewitt) [inline packages/addons/README.md]: #add-on package.mk skeleton

File: packages/addons/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+This is an example how the default structure of an addon should look like.
```
Comment (chewitt) [inline packages/addons/README.md]: This file shows a skeleton example of an add-on package.mk with content notes.

**Comment by chewitt** [comment]
It would be good to add explanations of the available functions() .. but it's no problem to add that later in a separate commit.

**Comment by CvH** [comment]
I add these changes (in a week), TX

**Comment by CvH** [comment]
@chewitt added your changes (fixup) - I squash if you :+1:

**Comment by CvH** [comment]
@chewitt squashed and removed empty line

**Pull request merged**